static/scenes/snowdodgeball/index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<!--
+ Copyright 2025 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lobster|Google+Sans:400,700" />
+  <link rel="stylesheet" href="index.css" />
+</head>
+
+<body>
+
+  <div id="module-snowdodgeball">
+    <!-- Start Screen Overlay -->
+    <div id="start-screen">
+      <h1 class="start-screen__title">Snow Dodgeball</h1>
+
+      <div class="start-screen__content">
+        <h2 class="instructions-title">Instructions</h2>
+
+        <div class="instruction-panels">
+          <div class="instruction-panel">
+            <img src="img/click-dodge-instruction.png" alt="Click to dodge" class="panel-image">
+            <p>Click an elf to dodge snowballs</p>
+          </div>
+          <div class="instruction-panel">
+            <img src="img/click-throw-instruction-large.jpg" alt="Throw snowball" class="panel-image">
+            <p>Click to throw snowballs</p>
+          </div>
+        </div>
+      </div>
+
+      <button id="start-button" class="start-button">START</button>
```

Comment:
These buttons will need translations (I think we should already have strings you can use?) and will also need aria labels